include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -9,15 +9,33 @@
 
 namespace onnxruntime {
 
+/**
+* Default_Global refers to all the transformations which are required for 
+  corrcetness and are execution provider independent
```

Comment:
Few spelling issues. [https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater) is very helpful. 

correctness
Default_ProviderSpecific
optimizations